scss/index.css

```diff
@@ -1,3 +1,906 @@
+}
+
+.pastEvents .events .event:nth-child(1) {
+  background-image: url("../images/car1.jpg");
+}
+
+.pastEvents .events .event:nth-child(2) {
+  background-image: url("../images/car2.jpg");
+}
+
+.darkFooter {
+  background-color: #272a31;
+  color: white;
+}
+
+@media screen and (max-width: 769px) {
+  .mobileMenu {
+    display: block;
+  }
+
+  header img {
+    margin: 0;
+    width: 45px;
+  }
+
+  .mobileMenu img {
+    width: 20px;
+    margin: 0;
+  }
+
+  header {
+    background-color: rgba(255, 255, 255, 0.202);
+    justify-content: space-between;
+    box-shadow: none;
+    -webkit-backdrop-filter: blur(4px);
+    backdrop-filter: blur(4px);
+  }
+
+  header nav {
+    display: none;
+  }
+
+  .mobileItems {
+    display: flex;
+    position: fixed;
+    width: 100%;
+    height: 100vh;
+    background-color: rgba(255, 255, 255, 0.437);
+    -webkit-backdrop-filter: blur(10px);
+    backdrop-filter: blur(10px);
+    top: 0;
+    z-index: 20;
+    padding: 30px;
+  }
+
+  .mobileItems img {
+    width: 20px;
+    height: 20px;
+    margin-bottom: 25px;
+  }
+
+  .mobileItems ul {
+    list-style: none;
+  }
+
+  .mobileItems ul li {
+    font-size: 35px;
+    margin: 10px 0;
+    font-weight: 700;
+  }
+
+  .mobileItems ul li:nth-child(2) {
+    animation: textLoads 0.5s ease-in-out;
+  }
+
+  .mobileItems ul li:nth-child(3) {
+    animation: textLoads 0.8s ease-in-out;
+  }
+
+  .mobileItems ul li:nth-child(4) {
+    animation: textLoads 1.1s ease-in-out;
+  }
+
+  .mobileItems ul li:nth-child(5) {
+    animation: textLoads 1.4s ease-in-out;
+  }
+
+  .mobileItems ul li:nth-child(6) {
+    animation: textLoads 1.7s ease-in-out;
+  }
+
+  footer {
+    background-color: white !important;
+    color: black !important;
+    padding: 30px 0;
+    -moz-column-gap: 20px;
+    column-gap: 20px;
+  }
+
+  footer p {
+    opacity: 0.7;
+  }
+
+  .pastEvents > p {
+    margin: 20px 0 40px 0;
+    text-align: center;
+    width: 90%;
+  }
+
+  .logoDesign p {
+    width: 80%;
+  }
+
+  footer p:nth-child(2) {
+    display: none;
+  }
+
+  .logoDesign {
+    text-align: center;
+  }
+
+  .logoDesign h2 {
+    width: 80%;
+  }
+
+  .logoDesign p span {
+    display: none;
+  }
+
+  .logoDesign .logo {
+    width: 300px;
+  }
+
+  .logoDesign .logo img {
+    height: 150px;
+  }
+
+  .landingPage {
+    padding: 100px 0;
+    height: min-content;
+  }
+
+  .landingPage div {
+    width: 90%;
+  }
+
+  .featured h3 {
+    font-weight: 700;
+    font-size: 25px !important;
+  }
+
+  .mainProgram h3 {
+    font-size: 25px !important;
+  }
+
+  .landingPage div h3 {
+    font-size: 25px !important;
+  }
+
+  .landingPage div h1 {
+    font-size: 35px !important;
+  }
+
+  .landingPage div p {
+    width: 95% !important;
+    padding: 10px !important;
+    max-width: 450px;
+  }
+
+  .landingPage div h2 {
+    font-size: 25px !important;
+  }
+
+  .landingPage div h4 {
+    font-size: 16px !important;
```

Comment:
- Kindly, you used `!important` money times 👍 please don't use it as much as possible to follow the best practice 👍 check best practices for [more info](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) 👍👍

Reply:
That is because, without using importnat, the css was not working in mobile version and i have to use important for that